src/jdk.crypto.ec/share/classes/sun/security/ec/ECKeyPairGenerator.java

```diff
@@ -116,7 +122,7 @@ public void initialize(AlgorithmParameterSpec params, SecureRandom random)
         this.random = random;
     }
 
-    private static void ensureCurveIsSupported(ECParameterSpec ecSpec)
+    public static void ensureCurveIsSupported(ECParameterSpec ecSpec)
```

Comment:
This should not be public (package-private will suffice).

Reply:
Changed to package level.